examples/dynamics/rotating_pendulum.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+
+from sympy import (
+    symbols,
+    Function as SymFunction,
+    cos,
+    sin,
+    solve,
+    Eq,
+    reduce_inequalities,
+)
+from sympy.physics.units import acceleration_due_to_gravity
+from symplyphysics import (
+    Vector,
+    cross_cartesian_vectors,
+    vector_unit,
+    dot_vectors,
+    print_expression,
+)
+from symplyphysics.definitions import (
+    harmonic_oscillator_is_second_derivative_equation as harmonic_def,
+    period_from_angular_frequency as period_def,
+)
+from symplyphysics.laws.dynamics.vector import (
+    relative_acceleration_from_force as motion_law,
+)
+from symplyphysics.laws.kinematic.vector import (
+    acceleration_of_transfer_between_relative_frames as tranfer_law,
+    centripetal_acceleration_via_cross_product as centripetal_law,
+)
+from symplyphysics.laws.kinematic import (
+    tangential_acceleration_of_rotating_body as tangential_law,
+)
+
+# Description
+## A physical pendulum consisting of a ball fixed on the end of a thin rigid rod can freely
+## oscillate around the horizontal axis A passing through the upper end of the rod. Axis A
+## is fixed rigidly on the geometric axis of a horizontal disk uniformly rotating around this
+## (vertical) geometric axis with angular velocity `w`. Thus, the plane of oscillation of the
+## pendulum rotates together with the disk with the same angular velocity `w`. Find the period of
+## small oscillations of the pendulum if the mass of the rod is negligible compared to the mass
+## of the ball. Under what condition will the lower vertical position of the rod become an unstable
+## equilibrium position?
+
+# Solving the problem in the non-inertial reference frame fixed to the plane in which the
+# pendulum oscillates.
+
+rod_length, disk_angular_velocity, time = symbols(
+    "rod_length disk_angular_velocity time",
+    positive=True,
+)
+
+ball_angle = symbols("ball_angle", real=True, cls=SymFunction)
+
+ZERO = Vector([0, 0, 0])
+
+# The y axis is parallel to the pseudovector of the disk's angular velocity and the x axis
+# runs from left to right in the plane where the pendulum's oscillations occur. The z axis
+# is perpendicular to that plane.
+
+disk_angular_velocity_vector = Vector([0, disk_angular_velocity, 0])
+
+ball_position_vector = Vector([
+    rod_length * sin(ball_angle(time)),
+    -1 * rod_length * cos(ball_angle(time)),
+    0,
+])
+
+gravity_force_vector = Vector([0, -1 * motion_law.mass * acceleration_due_to_gravity, 0])
+
+ball_acceleration_vector = motion_law.relative_acceleration_law(
+    force_=gravity_force_vector,
+    coriolis_acceleration_=ZERO,  # can be omitted since it is parallel to the z-axis
```

Comment:
I do not see how it can be omitted. Angular velocity is in Y axis and motion is in X axis. So coriolis acceleration is indeed in Z axis, but not zero.